Service/Host/CategoryController.cs

```diff
@@ -1,11 +1,24 @@
-﻿namespace Host
-{
-    using Model.SqlCe;
+﻿using AutoMapper;
+
+using TodoSystem.Model.Implementation;
+using TodoSystem.Model.SqlCe;
 
+namespace Host
+{
     /// <summary>
     /// Service for Category Controller.
     /// </summary>
-    public class CategoryController : SqlCeCategoryController
+    public class CategoryController : BaseCategoryController
     {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="CategoryController"/> class.
+        /// </summary>
+        public CategoryController()
```

Comment:
It was problem because out-of-the-box WCF doesn't support constructors with parameters. But there is solution. Also it will be possible to implement IoC.